eox_nelp/pearson_vue/validators.py

```diff
@@ -0,0 +1,71 @@
+"""Module to add validators related Pearson Vue Integration"""
+
+import logging
+
+from pydantic import ValidationError
+
+from eox_nelp.pearson_vue.data_classes import CddRequest, EadRequest
+
+try:
+    from eox_audit_model.decorators import audit_method
+except ImportError:
+    def audit_method(action):  # pylint: disable=unused-argument
+        """Identity audit_method"""
+        return lambda x: x
+
+
+logger = logging.getLogger(__name__)
+
+
+def validate_cdd_request(cdd_request):
+    """
+    Validates an CDD request dictionary using a Pydantic model.
+
+    This function attempts to create a Pydantic model instance (likely named `class CddRequest`:
+`)
+    from the provided `cdd_request` dictionary. It performs data validation based on the
+    model's data type definitions.
+    Then if there is an error then that error is raised using audit. audit_validation_error
+
+    Args:
+        cdd_request (dict): The dictionary containing the EAD request data.
+    """
+    try:
+        CddRequest(**cdd_request)
+    except ValidationError as validation_exception:
+        logger.info("Validation error for cdd_request: %s \n %s", cdd_request, validation_exception)
+        audit_validation_error(validation_exception.json(), cdd_request=cdd_request)
+
+    logger.info("Valid values for cdd_request: \n @clientCandidateID: %s", cdd_request["@clientCandidateID"])
```

Comment:
do you think this is necessary, I mean there are only two options, the error and the success if there is an error  this message wont' be shown, on the other hand, if there are no errors this message will appear but will we need this?, probably we will check the eox-audit records, if we check, IMO this is noise in the logs